WEATHERAPP/Components/Login/LoginPage.html

```diff
@@ -1,2 +1,31 @@
 <!DOCTYPE html>
```

Comment:
Add event-handler functions for the buttons and inputs

Reply:
resolved